PACKAGE.md

```diff
@@ -0,0 +1,106 @@
+# Building the package
+## TL;DR
+
+The package is only available at `obs://systemsmanagement:SCC/suseconnect-ng`
```

Comment:
You could recreate it from git by creating a new osc package, copy `_service` from git and then run `osc service manual run`. Except the changelog.